tests/migrate_repository.rs

```diff
@@ -403,6 +415,13 @@ async fn migrate_repository() {
                 .await
             );
         }
+
+        // Expect that CA3 publishes nothing in the embedded repo
```

Comment:
This comment says CA3 but the following code says CA1, is this correct?

Reply:
indeed copy/paste error from previous test. This will be reviewed as part of issue #480